### PR TITLE
Add more checks v2

### LINKED ISSUE #219: Cop cars does not work
As a police officer you are not able to buy cars. I belive that it checks if the officer has a driver license,(only suppost to do this for civs)

this was checked at the newest version of altislife

### PR DESCRIPTION
**Pull request by Jason2605**
<Please review the guidelines for contributing to this repository. The link is above.>

Resolves #219 

#### Changes proposed in this pull request: 
- Cars were still being checked on the license no matter the side, extra checks have been added to make sure side is not an issue.

### CODE REVIEW

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 1]
```diff
@@ -666,7 +666,7 @@ will modify the virtual space and the price of the vehicle, but other informatio
 
     class B_SDV_01_F {
         vItemSpace = 50;
-        conditions = "license_civ_boat || {license_cop_cg}";
+        conditions = "license_civ_boat || {license_cop_cg} || {(playerSide isEqualTo Independent)}";
```
Comment (BoGuu) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: `Independent `

Not sure if case will be an issue here

Comment (Jason2605) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: /shrug ill change it

**Comment by BoGuu** [comment]
LGTM

**Comment by WolfSkin1337** [comment]
@Jason2605 can you delete the spaces at:
- https://github.com/AsYetUntitled/Framework/pull/225/commits/027b8fca8a2fa15b916687f4d1a3d988b127212c#diff-e5006880698faf45ea1f3c05cdc2f97aR674
anyone has add extra spaces.

And
missed ` || {(playerSide isEqualTo Independent)}`
- at https://github.com/AsYetUntitled/Framework/pull/225/commits/027b8fca8a2fa15b916687f4d1a3d988b127212c#diff-e5006880698faf45ea1f3c05cdc2f97aR677

**Comment by WolfSkin1337** [comment]
lgtm

**Comment by BoGuu** [comment]
LGTM

**Comment by Jawshy** [comment]
_Supersedes https://github.com/AsYetUntitled/Framework/pull/164._

**Comment by WolfSkin1337** [comment]
@Jason2605 
Do not forget to delete the file in your Branch, because we dont need this anymore if you integrate the Apex Content
https://github.com/AsYetUntitled/Framework/blob/MoreChecksV2/ApexContent/Config_Vehicles.hpp

**Comment by BoGuu** [comment]
@Jawshy Happy with this?

**Comment by Jawshy** [comment]
@BoGuu, ready to merge. Just need you to check again.

**Pull request merged**